module/zfs/vdev_raidz.c

```diff
@@ -1440,6 +1440,94 @@ vdev_raidz_child_done(zio_t *zio)
 	rc->rc_skipped = 0;
 }
 
+
+/* Rotating media load calculation configuration. */
+extern int zfs_vdev_mirror_rotating_inc;
+extern int zfs_vdev_mirror_rotating_seek_inc;
+extern int zfs_vdev_mirror_rotating_seek_offset;
+
+/* Non-rotating media load calculation configuration. */
+extern int zfs_vdev_mirror_non_rotating_inc;
+extern int zfs_vdev_mirror_non_rotating_seek_inc;
```

Comment:
I'll wait for performance data to see what tuning is relevant here. We might be fine with just looking at vdev queue size here. Maybe penalizing rotational vdevs by some factor would be enough.